src/lib/Scenes/Home/Components/CollectionsRail.tsx

```diff
@@ -36,6 +36,10 @@ const CollectionsRail: React.FC<Props & RailScrollProps> = (props) => {
     scrollToTop: () => listRef.current?.scrollToOffset({ offset: 0, animated: false }),
   }))
 
+  // if (!props.collectionsModule.results?.length) {
+  //   return null
+  // }
+
```

Comment:
I think you want this to be 
```suggestion
  if (!props.collectionsModule.results?.length) {
    return null
  }

```